libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -162,6 +162,13 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
     // @Units: m
     AP_GROUPINFO("ALT_MAX", 16, AC_PrecLand, _sensor_max_alt, 8),
 
+    // @Param: MOVING
+    // @DisplayName: Precision Landing Target stationary/moving
+    // @Description: Precision Landing Target stationary/moving
+    // @Values: 0:Stationary, 1:Moving
+    // @User: Advanced
+    AP_GROUPINFO("MOVING", 17, AC_PrecLand, _moving, 0),
```

Comment:
Let's change this to an OPTIONS parameter so we can shove more features in here in the future.